### PR TITLE
[jsk_perception] Support chainercv mask rcnn

### PR DESCRIPTION
**Pull request by knorth55**
- install mask rcnn in chainercv `0.13.0`

### CODE REVIEW

File: jsk_perception/node_scripts/mask_rcnn_instance_segmentation.py [hunk 1]
```diff
@@ -41,20 +54,37 @@ def __init__(self):
...
-        n_fg_class = len(self.fg_class_names)
-        self.model = chainer_mask_rcnn.models.MaskRCNNResNet(
-            n_layers=50,
-            n_fg_class=n_fg_class,
-            pretrained_model=pretrained_model,
-            anchor_scales=rospy.get_param('~anchor_scales', [4, 8, 16, 32]),
-            min_size=rospy.get_param('~min_size', 600),
-            max_size=rospy.get_param('~max_size', 1000),
-        )
+        self.model_name = rospy.get_param('~model_name', 'mask_rcnn_resnet50')
```
Comment (YutoUchimi) [inline jsk_perception/node_scripts/mask_rcnn_instance_segmentation.py]: Please update document about ...
- `~fg_class_names`
- `~model_name`

**Comment by knorth55** [comment]
@k-okada  can you merge this?

**Comment by k-okada** [comment]
please add "version_gte" to package.xml, if this requires a specific
version, though it is not supported yet, it helps people, i think
https://github.com/ros-infrastructure/rosdep/pull/694
--
◉ Kei Okada



2019年11月3日(日) 22:46 Shingo Kitagawa <notifications@github.com>:

**Comment by knorth55** [comment]
@k-okada travis passed. can you merge this PR?

**Pull request merged**